example/javalib/android/2-linting/1-basic-linting/build.mill

```diff
@@ -0,0 +1,29 @@
+package build
+
+import mill._
+import mill.javalib.android.{AndroidSdkModule, AndroidLintModule}
+
+// Create and configure an Android SDK module to manage Android SDK paths and tools.
+object androidSdkModule0 extends AndroidSdkModule {
+  def buildToolsVersion = "35.0.0"
+}
+
+// Actual android application
+object app extends AndroidLintModule {
+  def androidSdkModule = mill.define.ModuleRef(androidSdkModule0)
+}
+
+/** See Also: app/AndroidManifest.xml */
+/** See Also: app/src/main/java/com/helloworld/app/MainActivity.java */
+
+/** Usage
+
+> ./mill show app.androidLintRun  # Display full path to the linting report in HTML
+".../out/app/androidLintRun.dest/report.html"
```

Comment:
Let's `> cat out/app/androidLintRun.dest/report.html` and match some strings that we expect to be inside that report. I assume your examples contain some lint warnings, so we should be able to match them as part of the `cat` output

Reply:
We should also use `sed` to fix the lint warnings and run lint again to verify that no more warnings exist. This applies to the `2-custom-linting` setup as well